goldens/by_regex.err

```diff
@@ -0,0 +1 @@
+WRN by_regex cannot be used with ignore_prefixes (consider adding a non-capturing group to the start of your regex instead of ignore_prefixes: "(?:foo|bar)") line=36
```

Comment:
I forget offhand, is it even possible to cover the case of the regex failing to parse? That case panics, so perhaps it isn't feasible to test. But if it can be tested, we should do so. We can expect that users will hit that quite a lot, actually. It's a meme how much people mess up regexes lol.

Reply:
It is possible! That case doesn't panic, it prints a warning on the offending line. Done